docs/quickstart.md

```diff
@@ -223,26 +220,73 @@ Now that we have the script we can proceed to download the ISO and inject our co
 
 This will generate an ISO on the current directory with the name `elemental-teal-<ARCH>.iso`
 
-:::info
-The script uses the iso for the arch based on the system is being run from. If you want to cross build for another system,
-you can set the `ARCH` environment variable to the desired target system (x86_64, aarch64) and the iso will be build for that architecture.
-:::
+</TabItem>
+</Tabs>
```

Comment:
This just dropped the create iso script. Do we want to drop it completely? Or do we want to keep it and make clear that there is 2 methods to create the iso via script? IMO, I would drop it fully, use only the inject one AND add an extra tab to explain how to build the iso FROM the repo, making sure we explain that is only meant for DEVELOPMENT, trying new stuff, etc...

Or even create a new section for that? While I think we should point as much thinhgs as possible to use th official OBS images, I think its important to explain how you can build your own iso from scratch and add stuff on it, test new things and so on, so building from the repo with `make build && make build_iso`

Yeah, maybe an extra section for building iso or developments is needed to explain that.

Reply:
I think it's fine to drop it from the quickstart. It might fit better in the "customize the OS" page?